communities/epfl-hackers/community-assessment.md

```diff
@@ -0,0 +1,87 @@
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in working on cool tech projects outside of academia at EPFL.
+  
+### Size
+- How many people are currently in your community?
+  - Hard to tell as there is no official club or society, but I would estimate we are around 5 people.
+- How many would you _like_ to be in your community?
+	- Hard to say, but I would estimate that there are around 150 students (based on the number of EPFL students who applied to participate in LauzHack 2016) who would be interested in being part of such a community. That would be pretty great!
+- Historically, has the group always been about the same size? If not, what brought on that change?
+	- Can't really tell as there has been no metric. Many students are working on cool tech projects, but there hasn't really been any gathering to form a group.
+
+### Resources
+- What is currently available to people in your community to learn about tech? Are there meetups, seminars, online tools, office hours, hackathons?
+	- Outside of lectures, not alot.
+	- I organzied the first hackathon at EPFL last year - LauzHack
+	- There are colloquiums organized by the CS department at EPFL where guest lecturers present their research followed by a Q&A, although these are not very frequent and have fairly low attendance.
+	- Around "recruiting season" several companies come to EPFL and have some talks and workshops.
+
+### Accessibility
+- How accessible are the current resources in your community?
+	- Quite accessible - pretty much anyone can join in.
```

Comment:
Same comment as first - that's the perception from the inside, how does it look from outside? With no knowledge of your group or work, how would I find it?